### PR TITLE
Unit tests working and passing

### PR DESCRIPTION
**Pull request by MorrisonAngus**
Not verified against newest version of main. Please check before approving

### CODE REVIEW

**Review by lgrymaloski — APPROVED** [review]
Looks good!

File: elastic-layer/UnitTests/test_RecommendTests.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
...2024-01-01T08:00:00', 'charity': True}, {'listing_id': 26, 'seller_id': 6, 'title': 'TV', 'price': 20.0, 'location': {'latitude': 48.46, 'longitude': -123.31}, 'address': '987 Shine Avenue, Victoria, BC', 'status': 'AVAILABLE', 'listed_at': '2024-01-01T08:00:00', 'last_updated_at': '2024-01-01T08:00:00', 'charity': True}, {'listing_id': 39, 'seller_id': 6, 'title': 'Green Lamp', 'price': 20.0, 'location': {'latitude': 48.46, 'longitude': -123.31}, 'address': '987 Shine Avenue, Victoria, BC', 'status': 'AVAILABLE', 'listed_at': '2024-01-01T08:00:00', 'last_updated_at': '2024-01-01T08:00:00', 'charity': True}, {'listing_id': 40, 'seller_id': 6, 'title': 'Side Lamp', 'price': 20.0, 'location': {'latitude': 48.46, 'longitude': -123.31}, 'address': '987 Shine Avenue, Victoria, BC', 'status': 'AVAILABLE', 'listed_at': '2024-01-01T08:00:00', 'last_updated_at': '2024-01-01T08:00:00', 'charity': True}, {'listing_id': 41, 'seller_id': 6, 'title': 'Purple Lamp', 'price': 20.0, 'location': {'latitude': 48.46, 'longitude': -123.31}, 'address': '987 Shine Avenue, Victoria, BC', 'status': 'AVAILABLE', 'listed_at': '2024-01-01T08:00:00', 'last_updated_at': '2024-01-01T08:00:00', 'charity': True}]:
```
Comment (haleyrutten) [inline elastic-layer/UnitTests/test_RecommendTests.py:73]: Please put in .json file and read from it.

File: elastic-layer/UnitTests/test_SearchTests.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+def test_listingInDatabaseSearch():
+    loadListings()
+    loadUsers()
+    # using testing data loaded into the elastic database,
+    # Preform a search for a specific item ie. "Bike"
+    search_results = search.searchVikeandSell(elastic_client, "listing", "Bike")
+
+    # and Assert that the listing returns the listing that is ensured to be in the database
+    test_passing = 1
+    expected_results = [{'listing_id': 7, 'seller_id': 7, 'title': 'Mountain Bike', 'price': 200.0, 'location': {'latitude': 48.429, 'longitude': -123.365}, 'address': '123 Bike Trail, Victoria, BC', 'status': 'AVAILABLE', 'listed_at': '2024-07-15T17:00:00', 'last_updated_at': '2024-07-15T17:00:00', 'charity': False},
```
Comment (haleyrutten) [inline elastic-layer/UnitTests/test_SearchTests.py:53]: json is our friend <3

**Review by haleyrutten — COMMENTED** [review]
make it prettier please <3

**Comment by MorrisonAngus** [comment]
I made it prettier
Seems like the no charity items user still receives charity items in a cold start recommendation

**Pull request merged**